distribution/src/bin/opensearch-service.bat

```diff
@@ -24,6 +24,10 @@ exit /B 1
 set OPENSEARCH_VERSION=${project.version}
 
 if "%SERVICE_LOG_DIR%" == "" set SERVICE_LOG_DIR=%OPENSEARCH_HOME%\logs
+rem The logs directory must exist for the service to start.
+if not exist "%SERVICE_LOG_DIR%" (
+	mkdir "%SERVICE_LOG_DIR%"
```

Comment:
@burck1 do you think we should be using `mkdir -p` instead to create non-existing parent folders (if any)?

Reply:
Hi @reta. That is a good thought. Fortunately, since this is a windows batch script, the windows variant of mkdir creates full directory trees by default, so the -p parameter is not necessary.